8.0/docker-entrypoint.sh

```diff
@@ -143,7 +143,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			--  or products like mysql-fabric won't work
 			SET @@SESSION.SQL_LOG_BIN=0;
 
-			DELETE FROM mysql.user WHERE user NOT IN ('mysql.session', 'mysql.sys', 'root') OR host NOT IN ('localhost') ;
+			DELETE FROM mysql.user WHERE user NOT IN ('mysql.infoschema', 'mysql.session', 'mysql.sys', 'root') OR host NOT IN ('localhost') ;
```

Comment:
Was this released?
```
mysql> SELECT user FROM mysql.user;
+---------------+
| user          |
+---------------+
| root          |
| mysql.session |
| mysql.sys     |
| root          |
+---------------+
4 rows in set (0.01 sec)

mysql> SELECT VERSION();
+-----------+
| VERSION() |
+-----------+
| 8.0.11    |
+-----------+
1 row in set (0.00 sec)
```

Reply:
No, instead the logic for removing unwanted users was removed from 5.7 and 8.0 completely (it was only needed for 5.5 and 5.6). But if you are upgrading from a version where the user wasn't present (either because it was an older version or it was mistakenly deleted) it won't be automatically recreated. Have you run mysql_upgrade?

Reply:
I think I've made mistake with `docker-compose` and didn't dropped old volume. Works now.